### DISCUSSION TITLE
Fade-in and fade-out while pausing during the level

### DISCUSSION THREAD

**Discussion by ScriptForAll** [SuperTux/supertux#2384] [Ideas]
The title says it all: I think while pausing during a level the music should not stop. It feels like a sudden break of mood, especially if you are just tweaking the settings during a playthrough.
 
I was playing the "Industrial Island" wordmap (amazing level design btw) and felt super immersed into the game. I wanted to change one simple keybinding and suddenly, after pausing, the music stops.
I can't be the only one that thinks that this is a complete mood killer, right? 

What y'all think?

**Comment by provigz** [comment]
To me music pausing when the game is paused seems like the proper behaviour. When you pause the game, you expect it to stop entirely, not just some parts of it.
